pulumi/tests/test_network.py

```diff
@@ -38,9 +38,9 @@ def test_number_of_subnets(self):
 
         def check_number_of_subnets(args):
             public_subnets, private_subnets = args
-            self.assertEqual(len(public_subnets), 2, f'There should be 2 public subnets since desired_az_spread is 2')
-            self.assertEqual(len(public_subnets), len(private_subnets),
-                             f'There should be the same number of public subnets and private subnets')
+            assert len(public_subnets) == 2, f'There should be 2 public subnets since desired_az_spread is 2'
```

Comment:
No reason to have these messages be formatting strings (`f"..."`), since there are no parameters.